eng/common/TestResources/New-TestResources.ps1

```diff
@@ -200,19 +200,29 @@ try {
         # Make sure the user is logged in to create a service principal.
         $context = Get-AzContext;
         if (!$context) {
-            Log "You are not logged in; connecting to 'Azure SDK Developer Playground'"
+            $subscriptionName = $SubscriptionId
+            $wellKnownSubscriptions = @{
```

Comment:
Do we not want to look this up? Takes about 6 seconds with no cache, and .8 seconds with a cache at least on my box:

```
$ (Get-AzSubscription -SubscriptionId "faa080af-c1d8-40ad-9cce-e1a450ca5b57").Name
Azure SDK Developer Playground
```

Reply:
Alternatively, you could log "connecting to ID" before, and after login, get the name out of the context.

Reply:
This only runs if you're not logged in. You can't get sub info if you're not:

```powershell
PS> Get-AzSubscription -SubscriptionId $env:AZURE_SUBSCRIPTION_ID
Get-AzSubscription: Run Connect-AzAccount to login.
```

Reply:
...@heaths has informed me of the fact that this obviously won't work when you aren't yet logged in :)